examples/server_profile_template.rb

```diff
@@ -59,4 +59,5 @@
 oneview_server_profile_template 'ServerProfileTemplate2' do
   client my_client
   action :delete
+  only_if { client[:api_version] == 500 && client[:api_version] == 600 }
```

Comment:
change && to ||
Since ```client[:api_version]``` variable cannot have 2 different values at same time and this && operator can never be satisfied